renderer/configuration/README.md

```diff
@@ -0,0 +1,85 @@
+# Configuration File
+
+Our goal for Robotouille is for it to be configurable so that users are able to use their own assets and customize the predicates to their liking. We have created a simple json for you to configure the game.
+
+## Configuring the Game
+
+### Description
+
+There are four main types of objects in the game. Each object has a set of properties that can be configured. The features we currently support are:
+
+- **Player**: changing the front, back, left and right image for the player
+- **Floor**: changing the floor image
+- **Item**: changing universal scaling and offset constants, and adding items that can each have their own offset constant and multiple images with corresponding predicates
+- **Station**: adding stations and their corresponding images
+
+### Example Format
+
+Here we describe the format of the JSON files under ```configuration```.
+
+```
+{
+    "version" = The current version of the configuration file
+
+    "player" = A list of player models that can be used in the game. Currently, robotouille is single-player ONLY. So the name of the player cannot be changed. 
```

Comment:
Robotouille

Reply:
I don't understand why "robot" can't be changed because Robotouille is single-player. There would be no difference if Robotouille were multiplayer - everyone can have robot assets. There is no difference here if we have singleplayer or multiplayer, only if we add a new player asset.